### DISCUSSION TITLE
MAKING CODES EASY AND ACCESSIBLE FOR BEGINNERS

### DISCUSSION THREAD

**Discussion by Dorane-Star** [Cyfrin/foundry-full-course-cu#3036] [Ideas]
Hi Patrick and Your Team, Great Job!

i would like to ADVICE all of your team that, it could be good if you could have attached the codes to each of its lessons rather than providing them in general, as this creates confusion and errors for beginners.

Example of what i mean: If you take a look at  " [Solidity 101 Section 3: Remix Fund Me]"....it contains several lessons. Now for my case, i am at lesson {More Solidity Math} where the functions taught are ['getPrice' and 'getConversionRate'].

PROBLEM ENCOUNTERED: OTHER CODES NOT ASSOCIATED WITH THE TAUGHT FUNCTIONS CONFUSE ME.

SOLUTION ADVICED: IT IS BETTER THE CODES ONLY  ASSOCIATED WITH THE TWO AFORE-MENTIONED FUNCTIONS APPEAR NEXT TO THEIR LESSON, OTHERWISE IF A BEGINNER WANTS TO CODE FOR A SPECIFIC LESSON, THEN THE REST OF OTHER CODES CREATES ERRORS AND CONFUSIONS AS THEY ARE NOT ASSOCIATED WITH THOSE FUNCTIONS.

**Comment by EngrPips** [comment]
Hello @Dorane-Star, Thanks for this awesome feedback. I trust @PatrickAlphaC  will address this appropriately as soon as possible.

**Comment by PatrickAlphaC** [comment]
I see what you're saying! Thanks for the suggestion! We will take this into account.
